zebra-state/src/request.rs

```diff
@@ -234,6 +235,18 @@ pub struct ContextuallyVerifiedBlock {
     pub(crate) chain_value_pool_change: ValueBalance<NegativeAllowed>,
 }
 
+/// Data related to an invalidated block including the [`ContextuallyVerifiedBlock`] and
+/// descendants.
+#[derive(Clone, Debug)]
+pub struct InvalidatedBlockData {
+    /// The block that was invalidated.
+    pub block: ContextuallyVerifiedBlock,
+    /// All descendant blocks that were also removed in order by [`block::Height`].
+    pub descendants: BTreeMap<block::Height, ContextuallyVerifiedBlock>,
+    /// Timestamp in which the block was invalidated.
+    pub timestamp: SystemTime,
```

Comment:
What's the intended purpose of this timestamp?

Reply:
I added the timestamp in case we wanted to report the time of the invalidation during the RPC call. It's really unnecessary and will be removed with the removal of the InvalidatedBlockData struct